setup.sh

```diff
@@ -56,8 +56,13 @@ create_frontend_properties() {
     if [[ ! -d "$path" ]]; then
       mkdir "$path"
     fi
-
-    aws s3 cp --profile frontend s3://aws-frontend-store/template-frontend.properties "$path/$filename"
+    
+    if [[ "$(fdesetup status)" != "FileVault is On." ]]; then
+      echo your hard disk is not encrypted! follow these instructions: https://support.apple.com/en-gb/HT204837
```

Comment:
you can use the `EXTRA_STEPS` thing here, have a look at the other examples in the file